packages/xdl/src/Analytics.ts

```diff
@@ -87,6 +90,15 @@ export class AnalyticsClient {
   }
 }
 
-const defaultClient = new AnalyticsClient();
+class XDLClient extends AnalyticsClient {
```

Comment:
So the goal is to _continue_ sending all events to the current XDL segment source, but add an additional segment source for specific kinds of events.  the snippet you linked only writes to one source, so it would not work for this requirement.

Reply:
did not pull the second instance out into a new file because i'd have to export the analytics client class & potentially those global variables... It's simpler to just embed it in xdl/analytics, but can make the change if you feel strongly